easybuild/easyconfigs/g/Grace/Grace-5.1.25-foss-2018a-5build1.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'Grace'
+version = '5.1.25'
+versionsuffix = '-5build1'
+
+homepage = 'http://plasma-gate.weizmann.ac.il/Grace/'
+description = """Grace is a WYSIWYG tool to make two-dimensional plots of numerical data."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['ftp://plasma-gate.weizmann.ac.il/pub/grace/src/grace5/']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['%(name)s-%(version)s%(versionsuffix)s.patch']
```

Comment:
@hajgato that patch set is a bit of a monster. Any chance of providing a brief summary of why they're required?

Reply:
@verdurin `Grace` uses some ancient libraries, for example `FFTW 2` (from 1999). First I was looking for a solution to use `FFTW3`, and I found the solution of `ubuntu`. The main part of the patch is the `T1LIB` patch, which is bundled with `Grace`, and it is from 2007 (a better alternative would be `freetype2` or `ImageMagick`). I am not sure whether all the patches are needed, but with those patches we have exactly the same version of ubuntu's `5.1.25-5build1`. Should I insert the whole story in the header of the patch?

Reply:
@hajgato Providing a bit more context in the patch would make sense indeed, since people may wonder what the deal is with this huge patch later...

Reply:
@boegel @verdurin More context provided in the patch.